openquake/fnm/exporter.py

```diff
@@ -72,6 +73,39 @@ def _get_profiles(kite_surf):
     return profiles
 
 
+def make_fault_geojson_feature(fault, skip_props=('trace'), z_unit='m'):
```

Comment:
These functions do not have a description, though it should be intuitive to undertand what they do